### PR TITLE
Assets 98992- Campaign details/overview

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Start implementing campaign details. The 'overview' tab and 'deliverables' tabs are basically done. Still need to implement tab-switching functionality.

- Before: https://main--assets-distribution-portal--adobe.hlx.page/sample-public-site
- After: https://assets-98992--adobe-gmo--hlxsites.hlx.page/drafts/mdickson/campaign-details

### CODE REVIEW

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+    `;
+    const tableRoot = block.querySelector('.table-content');
+    tableRoot.appendChild(rows);
+    decorateIcons(block);
+}
+function buildRows(data) {
+    console.log(data);
+    const rows = document.createElement('div');
+    const uniqueCategories = getUniqueValues(data, 'category');
+    uniqueCategories.forEach((category) => {
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:458]: can you add a condition to check if `category` and `subcategory` are not present then just show the deliverables without grouping. Something like this 
image

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+            <div class="table-wrapper">
+                <div class="table-header">
+                    <div class="header table-column column1">Content Name</div>
+                    <div class="header table-column column2">Content Type</div>
+                    <div class="header table-column column3">Channel</div>
+                    <div class="header table-column column4">Review Link</div>
+                    <div class="header table-column column5">Final Asset</div>
+                    <div class="header table-column column6">KPI</div>
+                    <div class="header table-column column7">Status Update</div>
+                    <div class="header table-column column8">Due Date</div>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:439]: Update label to "Completion Date"

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+                    </div>
+                </div>
+                <div class="total-assets">
+                    <div class="h3">Total Assets</div>
+                    <span class="description">7</span>
+                </div>
+            </div>
+            <div class="table-wrapper">
+                <div class="table-header">
+                    <div class="header table-column column1">Content Name</div>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:432]: Update label to "Deliverable Name"

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 4]
```diff
@@ -0,0 +1,526 @@
...
+                    <div class="header table-column column2">Content Type</div>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:433]: Update label to "Deliverable Type"

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 5]
```diff
@@ -0,0 +1,526 @@
...
+                    <div class="card-heading h3">Audiences</div>
+                    <div class="audience card-content">
+                        <span class="icon icon-gear"></span>
+                        Existing Express Users
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div id="tab2" class="deliverables">
+            <div class="page-heading">
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:416]: comment out "Links to Important Artifacts" section for MVP

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 6]
```diff
@@ -0,0 +1,526 @@
...
+                <div class="kpis-wrapper">
+                    <span class="h3">KPIs to Measure Success</span>
+                    <ul>
+                        <li>PR impressions & dedicated earned stories</li>
+                        <li>Mobile exports</li>
+                        <li>Community & social interactions</li>
+                        <li>100% by EOL</li>
+                    </ul>
+                </div>
+                <div class="use-cases-wrapper">
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:364]: Comment out "Hero use case" and "Links to Important Artifacts" for MVP

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 7]
```diff
@@ -0,0 +1,526 @@
...
+                <div class="header-row2">
+                    <span class="icon icon-calendar"></span>
+                    <span class="campaign-date">03/07/2024</span>
+                </div>
+            </div>
+        </div>
+        <div class="tab-wrapper">
+            <div id="tab1toggle" class="tab active">Overview</div>
+            <div id="tab2toggle" class="tab">Deliverables</div>
+            <div id="tab3toggle" class="tab ">Calendar</div>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:343]: Hide "Calendar for MVP"

File: blocks/gmo-campaign-details/gmo-campaign-details.js [hunk 8]
```diff
@@ -0,0 +1,526 @@
...
+        </div>
+        <div id="tab1" class="two-column overview inactive">
+            <div class="overview-wrapper">
+                <span class="h1 overview-heading">At a Glance</span>
+                <span class="h3">Strategy</span>
```
Comment (shiv-gup) [inline blocks/gmo-campaign-details/gmo-campaign-details.js:348]: update it to "Product Value"

File: blocks/gmo-campaign-details/gmo-campaign-details.js (hunk 6 above)
Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:364]: Is it okay if I hide them with CSS?

File: blocks/gmo-campaign-details/gmo-campaign-details.js (hunk 1 above)
Comment (mdickson-adbe) [reply blocks/gmo-campaign-details/gmo-campaign-details.js:458]: Yes, I'm noting a to-do right now because I'll need to think about how I want to do it.

**Pull request merged**